_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -65,11 +68,12 @@ Feel free to read the guide from start to finish or skip around to whatever part
 [[core_concepts]]
 == Core concepts
 
-The CIS AWS Foundations Benchmark is organized into four sections:
+The CIS AWS Foundations Benchmark is organized into five sections:
  +
  +
 
 . Identity and Access Management
+. Storage
```

Comment:
Suggestion for ease of navigation: Is it worth making everything in this bullet list a link to the section below? That said, I'm not sure which makes most sense. 

One option:
Right here, link to the `Recommendation Sections` which has a summary:

- ` - link:https://github.com/gruntwork-io/gruntwork-io.github.io/blob/ca410b41956807bc9f2cb1b71c9fbe2f839eb1ef/_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc#identity-and-access-management[Identity and Access Management]`

Then in that section ^ (under Production-grade Design), make the header itself a link to the main section:
- `link:https://github.com/gruntwork-io/gruntwork-io.github.io/blob/ca410b41956807bc9f2cb1b71c9fbe2f839eb1ef/_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc#identity-and-access-management-1[Identity and Access Management]`

Everything is linked at the TOC level, but you have to scroll to the top of the page to get there.

Reply:
Or, just link to the main section from here?

Reply:
@rhoboat hi Rho sorry for the delayed reply! I think I understand your point, but because this guide is directly consumed on the [Gruntwork marketing site](https://deploy-preview-392--keen-clarke-470db9.netlify.app/guides/compliance/how-to-achieve-cis-benchmark-compliance/#what-youll-learn-in-this-guide), does that left side-bar actually satisfy your point? As in I'm typically using that myself vs scrolling to the top of the page.

Reply:
hi @rhoboat gentle bump to resolve this one as we approach the finish line!

Reply:
Hey, so just confirming - is this something that we can update later on? I guess the impact would be small, if we just published the guide as is - and come back with the CIS Service Catalog work to re-do a few things (we could bring back up the discussion then)

Reply:
Yes, I don't feel it's blocking. Let's resolve it for now and revisit this issue later if need be.